src/App.jsx

```diff
@@ -14,70 +17,68 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
-
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+export const App = () => {
+  const [selectedGood, setselectedGood] = useState('Jam');
```

Comment:
You already have the `Jam` value inside the array 
```suggestion
  const [selectedGood, setselectedGood] = useState(goods[8]);
```